.github/workflows/backend.yml

```diff
@@ -86,37 +62,56 @@ jobs:
           client/python/requirements.txt
```

Comment:
Do we still need lines 60-64?

Reply:
I think so. From what I read here: https://github.com/actions/setup-python#caching-packages-dependencies, this caches the package download (not the install), so it can still be useful when we have to rebuild the env when there's a cache miss.